dice.toml

```diff
@@ -0,0 +1,29 @@
+[Server]
+Addr = '0.0.0.0'
+Port = 7379
+KeepAlive = 300
+Timeout = 300
+MaxConn = 0
+ShardCronFrequency = 1000000000
```

Comment:
Why have we set cron freq to such a high number? Is this to prevent timeouts while serving requests?

Reply:
This value is in nanosecond which resolves to 1 second, have added comments.

Reply:
Got it, in that case maybe the value should be higher. We would want the cron to run once every 30s instead of every second.
Otherwise it can cause massive performance issues.